packages/@uppy/react/package.json

```diff
@@ -21,11 +21,16 @@
     "url": "git+https://github.com/transloadit/uppy.git"
   },
   "dependencies": {
-    "@uppy/utils": "workspace:^"
+    "@uppy/utils": "workspace:^",
```

Comment:
Is this what you want here?

Reply:
No, I meant the version actually. Would that still look like that when deployed to npm? Or is this some Lerna thing?